apps/condo/domains/news/schema/NewsItemSharing.js

```diff
@@ -116,7 +123,11 @@ const NewsItemSharing = new GQLListSchema('NewsItemSharing', {
             if (operation === 'update') {
                 const oldStatus = existingItem.status
                 const newStatus = resolvedData.status
-                if (newStatus && !ALLOWED_TRANSITIONS[oldStatus].includes(newStatus)) {
+
+                if (!ALLOWED_TRANSITIONS[oldStatus]?.includes(newStatus)) {
```

Comment:
Why you remove `newStatus &&` condition from `if`?
This block checks that `status` field is updated (`status` field existed in `resolvedData`) and it's forbidden status transition.